imageio_ffmpeg/_io.py

```diff
@@ -375,7 +413,7 @@ def write_frames(
     # ----- Prepare
 
     # Get parameters
-    default_codec = "libx264"
+    default_codec = get_available_h264_encoders()[0]
     if path.lower().endswith(".wmv"):
```

Comment:
We could rewrite this a bit so that if the codec is specified, we don't need to do the extra ffmpeg invokation.
```py
if not codec:
    if path.lower().endswith(".wmv"):
        codec = "msmpeg4"
    else:
        codec = get_available_h264_encoders()[0]

```

Reply:
Thank you this is cleaner.